openhtf/output/servers/station_server.py

```diff
@@ -0,0 +1,622 @@
+"""HTTP server serving station info and an Angular frontend / web GUI.
+
+This module is comparable to OpenHTF's output.web_gui module. Some differences:
+- Serves info about a single station only.
+- Designed to run in the same process as an HTF test, instead of using the
+      XML-RPC-based station API.
+"""
+
+import contextlib
+import hashlib
+import itertools
+import json
+import logging
+import os
+import re
+import socket
+import sys
+import threading
+import time
+import types
+
+import openhtf
+from openhtf.output.callbacks import mfg_inspector
+from openhtf.output.web_gui_server import pub_sub
+from openhtf.output.web_gui_server import web_gui_server
+from openhtf.util import conf
+from openhtf.util import data
+from openhtf.util import functions
+from openhtf.util import multicast
+from openhtf.util import timeouts
+import sockjs.tornado
+import tornado.web
+
+
+_LOG = logging.getLogger('chauffeurhtf.station_server')
```

Comment:
This is probably not the logger name you want.

Reply:
#727 Done